LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2020
+COPYRIGHT HOLDER: 2 Degrees Investing Initiative France
```

Comment:
Please confirm the word "France" -- I expected this to be 2DII in general.

Reply:
yes we need the word France